src/sentry/api/endpoints/organization_releases.py

```diff
@@ -466,16 +466,17 @@ def post(self, request: Request, organization) -> Response:
             result = serializer.validated_data
             scope.set_tag("version", result["version"])
 
+            # Get all projects that are available to the user/token
+            # Note: Does not use the "projects" data param from the request
             projects_from_request = self.get_projects(request, organization)
-            allowed_projects = {
-                k: project for project in projects_from_request for k in (project.slug, project.id)
-            }
+            allowed_projects = {project.slug: project for project in projects_from_request}
+            allowed_projects.update({project.id: project for project in projects_from_request})
```

Comment:
two loops smh

Reply:
not optimal enough

Reply:
smh i had something better before, what do u want me to do 😭

Reply:
how about:
```Python
allowed_projects = {}
for project in projects_from_request:
    allowed_projects[project.slug] = project
    allowed_projects[project.id] = project
```

Reply:
nice and simple

Reply:
nothing fancy

Reply:
traditional loops 🙄 , looks good to me

Reply:
kids these days 👴